src/DateObject.ts

```diff
@@ -36,7 +36,7 @@ export interface DateProperties {
 	year: number;
 }
 
-const days = [ null, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 ];
+const days = [ -1, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 ];
```

Comment:
Maybe we should add a comment here overall... because this is an odd array and has no documentation.  Would `NaN` be appropriate here instead of `-1`?